homeassistant/components/nordpool/sensor.py

```diff
@@ -67,6 +81,42 @@ def get_prices(
     return result
 
 
+def get_lowest_price(
+    data: DeliveryPeriodData, area: str
+) -> tuple[float, datetime, datetime]:
+    """Get the lowest price from the data."""
+    price_data = data.entries
+    lowest_price: float = price_data[0].entry[area]
+    lowest_start: datetime = price_data[0].start
+    lowest_end: datetime = price_data[0].end
+    for entry in price_data:
+        for _area, price in entry.entry.items():
+            if _area == area and price < lowest_price:
+                lowest_price = price
+                lowest_start = entry.start
+                lowest_end = entry.end
+
+    return (lowest_price, lowest_start, lowest_end)
+
+
+def get_highest_price(
+    data: DeliveryPeriodData, area: str
+) -> tuple[float, datetime, datetime]:
+    """Get the highest price from the data."""
+    price_data = data.entries
+    highest_price: float = price_data[0].entry[area]
```

Comment:
since those two are exactly the same save the comparison, I'm thinking if it makes sense to make it into one helper. Not sure though